<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="anurags-MacBook-Air.local" failures="1" tests="1" name="Default test" time="28.687" errors="0" timestamp="2023-08-26T18:49:59 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="21.287">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertFalse(Assert.java:79)
at org.testng.Assert.assertFalse(Assert.java:89)
at stepDefinition.FaceboolStepDef.it_should_show_the_error_message(FaceboolStepDef.java:60)
at âœ½.it should show the error message(file:///Users/anuragchaturvedi/git/ExtentReportConfig/src/test/resources/Features/facebook.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
